script/QA/QA_17.txt

```diff
@@ -31,7 +31,8 @@ As for Mr. Protagonist, just give%{n}
 [sha:41aa3449deab5f87c318b0a76fe108219adb55eb]{
 -- Page 101, Offset 43503.
 -- あと２、３回くらい、
-it another two or three times.
+// Reads better?
+it another two or three tries.
```

Comment:
I think the idea is more "after you've experienced a couple more dead ends they won't bother you as much", rather than retrying _this_ dead end